docs/airgap.md

```diff
@@ -0,0 +1,68 @@
+# Air gap scenario
+
+In some situations you may want to sync two Helm chart repositories without direct connectivity between them. 
+charts-syncer support this scenario via intermediate chart bundles.
+
+An intermediate chart bundle is just a tarball containing the original chart code plus the container images needed to deploy the chart.
+
+Below you can see how the whole process looks like:
+
+## Step 1: Save chart bundles
+
+In this first step charts-syncer needs access to the source charts repository. 
+
+Create a config file like the following and save it to `config-save-bundles.yaml`
+
+```yaml
+source:
+  repo:
+    kind: HELM
+    url: https://charts.trials.tac.bitnami.com/demo
+target:
+  intermediateBundlesPath: /path/to/local/output-dir
```

Comment:
I would use a real, usable, path (e.g. `.`, `/tmp`, or `./bundles` if we are pre-creating the output folder in the tool)